src/EventStore/EventStore.php

```diff
@@ -158,7 +157,7 @@ public function writeToStream($stream_name, WritableToStream $events, $expectedV
             )
         ;
 
-        $request->setHeader('ES-ExpectedVersion', intval($expectedVersion));
+        $request->setHeader('ES-ExpectedVersion', intval($expected_version));
```

Comment:
yes, we were inconsistent, I prefer camelCase while @nicolopignatelli $expected_version. @nicolopignatelli which rule should we follow? I would use the PSR one...